build.sbt

```diff
@@ -14,8 +14,9 @@ def commonSettings(projectName: String): Seq[sbt.Def.Setting[_]] = Seq(
   name := projectName,
   organization := "io.iohk",
   version := "3.2.1",
-  scalaVersion := "2.12.12",
+  scalaVersion := "2.13.4",
   // Scalanet snapshots are published to Sonatype after each build.
+  resolvers += Resolver.mavenLocal,
```

Comment:
Why do we need local maven repo resolver?

Reply:
I'll remove it, used when Scalanet was unavailable at repo...